com.unity.visualeffectgraph/CHANGELOG.md

```diff
@@ -94,6 +94,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - GPU hang on some initialize dispatch during dichotomy (platform specific)
 - Compilation error undeclared identifier 'Infinity' [Case 1328592](https://issuetracker.unity3d.com/product/unity/issues/guid/1328592/)
 - Exposed Parameter placement can be moved after sanitize
+- Eye dropper in the color fields kept updating after pressing the Esc key
```

Comment:
You should merge with master, this changelog entry is supposed to be in section `13.0.0` now